pkg/cmd/cli/cmd/project.go

```diff
@@ -66,7 +66,6 @@ func NewCmdProject(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.
 		Short:   "Switch to another project",
 		Long:    projectLong,
 		Example: fmt.Sprintf(projectExample, fullName),
-		Aliases: []string{"projects"},
```

Comment:
So far, the only test against an `oc projects` command was to make sure it returned the same `--help` output as `oc project`, I have updated this, but aside from running new and existing tests on it, I haven't found any conflicts